### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by Gireen**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/idea.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+name: Idea
+about: Single idea without concrete problem
```
Comment (DolceTriade) [inline .github/ISSUE_TEMPLATE/idea.md:3]: Suggestion

File: .github/ISSUE_TEMPLATE/idea-that-solves-a-problem.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+name: Idea that solves a problem
```
Comment (DolceTriade) [inline .github/ISSUE_TEMPLATE/idea-that-solves-a-problem.md:2]: Enhancement

**Review by DolceTriade — CHANGES_REQUESTED** [review]

**Comment by Gireen** [comment]
The intention here is to have both issues types contain idea cause there only difference is if there is a problem issue attached to them. 
So the tags make the difference and could also change one into another. Like if an related problem is not accepted as such or if a new problem can be solved by an already on its own existing idea.

**Comment by DolceTriade** [comment]
I'm not sure if there is a benefit to actually separating these two categories.

**Comment by Viech** [comment]
Ideas may be short and simple if they solve a problem but otherwise we don't want to track any shower thought someone's having.

**Comment by Gireen** [comment]
the difference would only be the automatic set label but i think that can also be done by hand since it requires some control anyway.

**Review by DolceTriade — APPROVED** [review]

**Pull request merged**